errbot/backends/slack.py

```diff
@@ -460,12 +461,17 @@ def send_message(self, mess):
             log.debug('Message size: %d' % len(body))
 
             limit = min(self.bot_config.MESSAGE_SIZE_LIMIT, SLACK_MESSAGE_LIMIT)
-            parts = self.prepare_message_body(body, limit)
-
-            for part in parts:
+            if len(body) > limit:
+                self.api_call('files.upload', data={
+                    'channels': [to_channel_id],
+                    'content': body,
```

Comment:
body is `self.md.convert(mess.body)` which is a slack dialect (slack_markdown_converter) for formatting the messages does the attachment still supports this formatting or you need to use the basic text formatter (text from https://github.com/errbotio/errbot/blob/master/errbot/rendering/__init__.py#L29) ?